src/emoji.ts

```diff
@@ -35,6 +35,9 @@ export const EMOTICON_TO_EMOJI = new Map<string, IEmoji>();
 
 export const getEmojiFromUnicode = unicode => UNICODE_TO_EMOJI.get(stripVariation(unicode));
 
+const isRegionalIndicator = (x: string): boolean =>
+    Array.from(x).length === 1 && x >= '\u{1f1e6}' && x <= '\u{1f1ff}';
```

Comment:
A comment for why we need `Array.from()` would be appreciated, and a link to some emoji doc somewhere that validates the range. With the comments, this should get wrapped in a closure w/ return statement.